crates/pathfinder/src/rpc/v01/api.rs

```diff
@@ -318,7 +318,7 @@ impl RpcApi {
         contract_address: ContractAddress,
         key: StorageAddress,
         block_id: BlockId,
-    ) -> RpcResult<StorageValue> {
+    ) -> RpcResult<Option<StorageValue>> {
```

Comment:
This is an RPC method - and here we need to return ZERO and not an option.

In StarkNet an empty / missing storage element is equivalent to it being set to zero. Which is also the reason we used zero internally, but this becomes confusing in other instances (hence this issue / PR).

But in this instance, we need to map our `None` back to ZERO again.

Reply:
Makes me wonder that we should have a test to catch this (`None`).

Reply:
Oops, got carried away wrapping things in `Option`. Reverted.